### ISSUE TITLE
About the loss function problem

### ISSUE THREAD

**Issue by Pefect96** [ExplainableML/ProbVLM#7]
In the function of https://github.com/ExplainableML/ProbVLM/blob/cb69f28b1ab23142a1c671e004b09b5cb5d8a204/src/losses.py#L81, `nn.L1Loss` is inconsistent with Eq.3 in the paper, and Eq. 3 does not include the `nn.L1Loss`.

Waiting for your help! @udion @sgk98

**Comment by bravePinocchio** [comment]
Why isn't this line of code being used instead of the twenty-ninth line of code? Beta is power in the paper.
https://github.com/ExplainableML/ProbVLM/blob/cb69f28b1ab23142a1c671e004b09b5cb5d8a204/src/losses.py#L28-L29
![image](https://github.com/user-attachments/assets/fc3250de-2162-4581-b8fc-20d61da25711)

**Comment by udion** [comment]
For stability during training, beta in power leads to unstable training hitting nans, so we used Taylor series approximation, explained in Appendix of the paper here: https://arxiv.org/pdf/2307.00398 (Eq. 16 in appendix A)
